### PR TITLE
Fixes #8864: Safe setup of the Rudder repo in the manual

### PR DESCRIPTION
**Pull request by amousset**
https://www.rudder-project.org/redmine/issues/8864

### CODE REVIEW

**Comment by amousset** [comment]
This doc uses the https://www.rudder-project.org/rpm-repos/rudder_rpm_key.pub and https://www.rudder-project.org/apt-repos/rudder_apt_key.pub that do not exist yet, but it would be convenient to have a invariant url for our keys.

File: 10_installation/10_install_server/snippets/apt_key.txt.snippet [hunk 1]
```diff
@@ -0,0 +1,16 @@
+We will first import our key into apt using the following command:
+
+----
+
+wget --quiet -O- "https://www.rudder-project.org/apt-repos/rudder_apt_key.pub" | sudo apt-key add -
+
+----
+
+Our key fingerprint is:
```
Comment (jooooooon) [inline 10_installation/10_install_server/snippets/apt_key.txt.snippet]: Can you add here something like "Please check that the fingerprint matches the one displayed by this command". ?

File: 10_installation/10_install_server/snippets/rpm_key.txt.snippet [hunk 2]
```diff
@@ -0,0 +1,16 @@
+We will first import our key into rpm using the following command:
+
+----
+
+rpm --import https://www.rudder-project.org/rpm-repos/rudder_rpm_key.pub
+
+----
+
+Our key fingerprint is:
```
Comment (jooooooon) [inline 10_installation/10_install_server/snippets/rpm_key.txt.snippet]: And here of course

File: 10_installation/11_install_agent/10_install_agent_debian.txt [hunk 3]
```diff
@@ -1,23 +1,8 @@
...
-If your HTTP Keyserver Protocol (11371/tcp) is blocked you can use an alternate command:
-
-----
-
-root@rudder-server:~# wget --quiet -O- "http://keyserver.ubuntu.com/pks/lookup?op=get&search=0x474A19E8" | sudo apt-key add -
-
-----
-
-Add Rudder project repository:
+Then add Rudder project repository:
```
Comment (jooooooon) [inline 10_installation/11_install_agent/10_install_agent_debian.txt]: Can you replace this with "Then add Rudder's package repository:" ?

File: 10_installation/10_install_server/snippets/rpm_key.txt.snippet [hunk 4]
```diff
@@ -0,0 +1,16 @@
+We will first import our key into rpm using the following command:
```
Comment (jooooooon) [inline 10_installation/10_install_server/snippets/rpm_key.txt.snippet]: Could you add here a very short phrase to explain why?

File: 10_installation/10_install_server/snippets/apt_key.txt.snippet [hunk 5]
```diff
@@ -0,0 +1,16 @@
+We will first import our key into apt using the following command:
```
Comment (jooooooon) [inline 10_installation/10_install_server/snippets/apt_key.txt.snippet]: Same here

**Comment by amousset** [comment]
Commit modified

**Comment by amousset** [comment]
Commit modified

**Comment by amousset** [comment]
OK, merging this PR

**Pull request merged**